src/igvfd/loadxl.py

```diff
@@ -22,6 +22,7 @@
     'cell_line',
     'tissue',
     'technical_sample',
+    'assay_ontology_term',
```

Comment:
I was unsure how we want to sort this. I placed it at the end because technically, none of the preceding objects utilize assay_ontology_term, but I realize it looks somewhat out of place being separated from sample ontology. Do we have a preference here?